docs/core/retrieval-actions.rst

```diff
@@ -0,0 +1,134 @@
+:desc: Use a retrieval model to select chatbot responses
+       in open source bot framework Rasa.
+
+.. _retrieval-actions:
+
+Retrieval Actions
+=================
+
+.. edit-link::
+
+.. warning::
+   This feature is experimental.
+   We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+   However, the functionality might be changed or removed in the future.
+   If you have feedback (positive or negative) please share it with us on the `forum <https://forum.rasa.com>`_.
+
+.. contents::
+   :local:
+
+About
+^^^^^
+
+Retrieval actions are designed to make it simpler to work with :ref:`small-talk` and :ref:`simple-questions` .
+For example, if your assistant can handle 100 FAQs and 50 different small talk intents,
+You can use a single retrieval action to cover all of these. 
+From a dialogue perspective, these single-turn exchanges can all be treated equally, so this simplifies your stories.
+
+Instead of having a lot of stories like:
+
+.. code-block:: story
+
+   ## weather
+   * ask_weather
+      - utter_ask_weather
+   
+   ## introduction
+   * ask_name
+      - utter_introduce_myself
+
+   ...
+
+
+You can cover all of these with a single story:
+
+
+.. code-block:: story
+
+   ## chitchat
+   * chitchat
+      - respond_chitchat
+
+
+Training Data
+^^^^^^^^^^^^^
+
+Like the name suggests, retrieval actions learn to select the correct response from a list of candidates.
+As with other NLU data, you need to include examples of what your users will say in your NLU file:
+
+.. code-block:: md
+
+   ## intent: chitchat/ask_name
+   - what's your name
+   - who are you?
+   - what are you called?
+
+   ## intent: chitchat/ask_weather
+   - how's weather?
+   - is it sunny where you are?
+
+First, all of these examples will be combined into a single ``chitchat`` intent that NLU will predict.
```

Comment:
Should we also mention about the `/` delimiter in all such intents and what is suffix after it?